src/layouts/BlogLayout.astro

```diff
@@ -114,11 +114,14 @@ article :global(p),
 article :global(ol),
 article :global(ul),
 article :global(h2),
-article :global(h3),
 article :global(figure) {
   margin-block-end: var(--space-s);
 }
 
+article :global(h3) {
```

Comment:
Visually, due to the difference in font size, having the same spacing is not as balanced as making this a little smaller.